lib/main.dart

```diff
@@ -28,6 +29,12 @@ void main() async {
     ),
   );
   await Injector.initialise();
+  SystemChrome.setSystemUIOverlayStyle(
+    const SystemUiOverlayStyle(
+      statusBarColor: Colors.transparent,
+      statusBarIconBrightness: Brightness.dark,
+    ),
+  );
```

Comment:
cannot we set it through theme?

Reply:
Brightness.dark is basically an enum

Reply:
abi k liye theek hai per bad mai yehi tang kry ga. khair proceeding to merge it.